1-org/modules/cai-monitoring/README.md

```diff
@@ -0,0 +1,71 @@
+# Cloud Asset Inventory Notification
+Uses Google Cloud Asset Inventory to create a feed of IAM Policy change events, then process them to detect when they include a gmail address and revert that IAM policy change using a Cloud Function.  The terraform directory contains all the code required to deploy the solution (including the cloud function).
```

Comment:
I think there's some leftover from the previous sample.
"detect when they include a gmail address" should be "detect when the IAM policy change grants certain sensitive roles (Owner, Editor, Organization Administrator, Network Administrator, Organization Firewall Policy Administrator)"

Reply:
Oh, it is... Soon I'll push this fix!